distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc

```diff
@@ -0,0 +1,76 @@
+:title: System Configuration Settings
+:type: installing
+:status: published
+:summary: Configure Single Sign On(SSO) page of installer.
+:project: ${branding}
+:order: 07
+
+.[[_configuring_sso]]Configure Single Sign On(SSO)
+****
+Configure Single Sign On method: SAML or OIDC.
+
+.SAML SSO
+Enter the URLs for the IdP metatdata and set other options.
+
+IdP Server::
+Configure ${BRANDING}'s' internal Identity Provider Server.
```

Comment:
✏️ Looks like a stray single quote there after the `s`.

Reply:
fixed